pkg/workloads/cortex/client/cortex/binary/__init__.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2020 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from os import path
+import base64
+import sys
+import subprocess
+from typing import List, Optional, Tuple, Callable
+from io import BytesIO
+from cortex.exceptions import CortexBinaryException
+
+MIXED_CORTEX_MARKER = "~~cortex~~"
+
+
+def run():
+    """
+    Runs the CLI from terminal.
+    """
+    try:
+        process = subprocess.run([get_cli_path()] + sys.argv[1:], cwd=os.getcwd())
+    except KeyboardInterrupt:
```

Comment:
This error was caught here to prevent throwing a python exception when user ctrl-c's during `cortex logs`. I think it is okay if throw a KeyboardInterrupt exception when executing the python code (likely when someone is running cortex in a Python REPL like a notebook).